resources/templates/base.html

```diff
@@ -1,15 +1,19 @@
 <!DOCTYPE html>
 <html>
     <head>
-    <meta charset="utf-8">
-    <title>Doctopus</title>
-    <link rel="stylesheet" href="/assets/styles/css/master.css">
-    <link rel="shortcut icon" href="/assets/images/doctopus-favicon.png">
-    <script src="//use.typekit.net/txl6yqy.js"></script>
-    <script>try{Typekit.load();}catch(e){}</script>
+        <meta charset="utf-8">
+        <title>Doctopus</title>
+        <link rel="stylesheet" href="/assets/styles/css/master.css">
+        <link rel="shortcut icon" href="/assets/images/doctopus-favicon.png">
+        <script src="//use.typekit.net/txl6yqy.js"></script>
+        <script>try{Typekit.load();}catch(e){}</script>
+        <script type="application/json" id="app-state"></script>
```

Comment:
This is where the very "airship-ui-y" page context gets dumped as JSON. It might be worth killing this in favor of loading it via request to better un-marry the front-end from the back.